lib/modules/dosomething/dosomething_campaign/dosomething_campaign.features.field_instance.inc

```diff
@@ -3209,17 +3215,18 @@ The title determines the URL of the page. This means that if you are changing th
     'entity_type' => 'node',
     'field_name' => 'title_field',
     'label' => 'Title',
-    'required' => TRUE,
+    'required' => 1,
```

Comment:
were `TRUE` and `FALSE` not working or did Features just change these automatically?